src/main/java/nextstep/subway/auth/ui/AuthenticationPrincipalArgumentResolver.java

```diff
@@ -25,7 +25,15 @@ public boolean supportsParameter(MethodParameter parameter) {
 
     @Override
     public Object resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer, NativeWebRequest webRequest, WebDataBinderFactory binderFactory) {
-        String credentials = AuthorizationExtractor.extract(webRequest.getNativeRequest(HttpServletRequest.class));
-        return authService.findMemberByToken(credentials);
+        HttpServletRequest request = webRequest.getNativeRequest(HttpServletRequest.class);
+        String credentials = AuthorizationExtractor.extract(request);
+        if (isEquals(request.getRequestURI(),"/paths")) {
+            return authService.findMemberByToken(credentials, true);
+        }
+        return authService.findMemberByToken(credentials, false);
```

Comment:
```suggestion
        boolean skipException = "/paths".equals(request.getRequestURI());
        return authService.findMemberByToken(credentials, skipException);
```

isEquals 메서드는 따로 구현할 필요는 없어보이네요
skipUrl이 더 생겨난다면 list로 묶어서 contains로 처리해도 되겠네요.

Reply:
😮.. 동의합니다 :)

해당 방법을 당시에 생각해내지 못했네요 :( 
코드 가독성 저하, 쓸데없는 코드 발생 이 되었군요!!

수정하겠습니다.

피드백 감사합니다🙏